docs/resources/copy.md

```diff
@@ -0,0 +1,28 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "cosign_copy Resource - terraform-provider-cosign"
+subcategory: ""
+description: |-
+  This copies the provided image digest cosign copy.
+---
+
+# cosign_copy (Resource)
+
+This copies the provided image digest cosign copy.
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `destination_repo` (String) The destination repository.
+- `source` (String) The digest of the container image to copy.
```

Comment:
Hmm, should it take a tag?  If the destination is a repo, then what tag is applied?

Reply:
My thinking was that a separate `oci_tag` would be responsible for tagging. `cosign` shouldn't think about tags at all, IMO.

I don't feel that strongly though, and supporting tags would match `cosign` CLI behavior.